Lab-4.ipynb

```diff
@@ -0,0 +1,701 @@
E6dOhgxYgRu3bqldFt1lMG2bdvQunVrGBsbw8rKCt26deONYFWpWbNmAICsrCy5zxhjuHjxIgIDA9G6dWteZ9vb26Nnz544ePBgqeknJSUhODgY7u7uvC5t1KgRhg4dqrROUmbz5s3Q1taGtrY2tmzZIvf5f//7X3Ts2BFmZmYwNzdHu3bt8McffwAA2rRpA0EQsGvXLpltSr7kuW3bNnh7e8PKygqCIMj9AvbgwQOMGzcOderUgYGBAaysrNCpUyeEhoYq7VZRmWtTut4pLCzEsmXL4ObmBiMjI1hbW6NPnz64efOmynK7cOECevbsCUtLS5iYmKBly5YICQkBY0zldmU1YsQIAFDaq0BcLq5Xmj179sDHxwc2Njb8Gho+fDhu376tcrvU1FQEBATA2dkZBgYGqF27NiZMmIDXr1+Xab83b97EqFGj+LG1tLSEt7c3Nm7ciKKiojKloRIrxZ49exgABoA9fPiwtNUVunfvHnN2dubp6OjoMGNjY/5/S0tLdv78ebntPv74YwaArVixgn300UcMADM2Nma6urp82969ezPGGFu3bh0TBIFpaWkxMzMz/rmenp7CtLdu3coAsI8//ljpfrdu3Sr3WVxcnMx3MTIyYkZGRvz/tWvXZg8ePFBYDtu2bWNaWlp8XQsLC6ajo8MAsH79+rGRI0cyAGzevHnlKl/GGNuwYYNc2tra2vz/fn5+rLCwkK//008/MXt7e15Wurq6zN7env9duHChTPsV86yoHEV2dnYMABs/fjxflpaWxvMm7t/U1FRm2aRJkxSmN2zYMAaABQUFsVatWvHjbG5uzgCws2fPMnt7e2ZgYMAAMBMTE5nv5uTkxNOKi4tjAJi2trbCfUVERDBLS0uZ80n6/Cq53YkTJxgAVr9+fYXp7dmzh+nr6/PtDQ0NmZ6eHv9/8+bN2Zs3b5SWpSKjR49mAJiNjQ3Lzc0t17bS1q1bp/IcGjlyJCsqKpLZpqCggH8eHx+vMF1VZdy+fXsGgP3888+sWbNmDAAzMDBghoaGPF0bGxv2+PFjvs2TJ0+Yvb09r0MMDQ1ljq+9vT179epVmb6zeLxU5V+RQYMGMQAsODiYtWjRQu4czMnJ4evu3LlT5pgbGRnJ1GHu7u4sOTlZbh/p6emsZcuWMueeeI2YmZmxHTt2MABMX19fblt7e3sGgF26dElh/nNycni6r1+/lvs8PDycfxdx3+L1BIDVrVtX4f1A3O/OnTuZk5MTr7Olz3EXFxeWkZGhMF9//PGH3PUhfa9o1KgRX3fz5s0MAKtZs6bceSlKSUnh6Z07d07hOors3btXpi4yNjaW+f66urps9+7dCretbBmMGzeOr6elpcUsLCyYIAhMEAS2fv36Uo+tMuI56+vrq3SdESNGMACsdevWcp8lJSWVWmdPnTpVadqnTp1iFhYWSuvSkufxL7/8ojS/y5cvV3kcZs+ezdMVBIFZWFjwum3mzJnMy8uLHyNl+xw/fjw/BpaWlkwQBHb06FG+7r59+2SOqbm5Ob+fA2A9evRg2dnZcnmrzLUpHsMFCxawzp0783KUvkaMjIxYZGSkwrQVxSBiHT906FCe/g8//KBwe2XEcnN2dmaFhYXM0dGRCYLAnjx5IrNeQUEBs7OzY4IgsGfPnrHPPvuMAWD+/v5yaRYUFLDBgwfL3Gulzx9tbW22efNmhfmJj49nderUUViPODo6sg0bNsjVJ9KWLVsmU04mJiYy98Ju3brJ3WtLq1NLKjX4fvToEU/Q19e33IFBeno6L4S+ffuyGzdusIKCAsZY8U10+PDhDACzt7dnaWlpMtuKQbC5uTlr1KgRrzzz8vLYxo0b+Ym+cOFCpqury4KCgngaT58+ZW3btmUAmKenp1y+KhJ85+Xl8QChVq1a7Pjx40wikTCJRMJOnjzJateuzQAwV1dXuQNz//59fqH27NmTn5TZ2dlszZo1TEdHh59Y5Q2+L1y4wE+UAQMG8CDi3bt37Pvvv2eCIDAA7LvvvitXOZRFacF3dnY23/+MGTP48oyMDNarVy+2a9cu9vLlS37zTEtLY2vWrOEXyv79++XSFINvExMTZmVlxfbs2cPy8/MZY4w9fvyYn1/ieoq+t0hVYBgbG8tvLi1btmQRERE8nykpKezYsWNswIABMtuoCr4vXbrEdHR0mI6ODgsKCmLPnz9njDFWWFjILl68yDw8PPj5UR7i9TVs2LBybSftzJkz/BwaNGgQe/HiBWOMsbdv37KFCxfyY1iyUlZX8G1hYcHq1avHjh8/zoqKilhRURE7ffo0q1GjBgPAhgwZIreteIMdO3Zshb93ZYNvExMTZm1tzcLCwvg5+OjRI36enD9/nmlrazMdHR02e/Zsvo+CggJ24cIFHlx/+umncvvw8/Pj+9ixYwc/r2/cuMFatGjBg2N1B99xcXHMxMSEAWATJ05kcXFx/JjcunWLde3alQHFDw0lg15xvxYWFszDw4NduXKFMcZYfn4+CwsL49fTnDlz5PIUERHBb3DdunVj169f558lJCSwffv2yTzAZ2Zm8vT+/vtvhd9z7dq1DABr2LChws+VOXbsGBs3bhyLiIhgKSkpjDHGJBIJe/LkCZs0aRIPqBXdZCtTBps2beLHZdasWSw9PZ0xxtjLly/Z4MGDmb6+Pn8IUGfwnZaWxpYuXcqv85JBKWOMpaamst69e7M9e/awV69e8WOfmprKVq1axR+YDx06JLft/fv3eZ3u4eHBzpw5I1OXHj16lH3++ecy2ygLvufMmcMDqr/++ktuX3/99RcvQ39/f5aUlMTzGRgYyARB4NeOsuDbxMSEaWlpsR9++IE/JKWlpfG07t+/z49D165dWVxcHGOs+Lr6+eef+cP1l19+KZc/dQTfFhYWzNbWlu3bt4/l5+cziUTCbty4wRspvb295dK9d+8ej0F69erFnj59yhgrvo5+/PFHmXKpTPDNGGPTp09XeO89fPgwA8A6derEGGMqg+958
```

Comment:
Agree no time-dependence. Yeah I think if we plot the difference in the images and then take the average of them we'd be able to find the background. Overall really nicely worded and I didn't see any typos like in my lab XD